.github/workflows/release-lapis2.yml

```diff
@@ -0,0 +1,70 @@
+name: Release LAPIS 2
+
+on:
+  push:
+  workflow_run:
+    workflows: [LAPIS v2]
```

Comment:
Do we want to run this also when `LAPIS v2 Docs` runs? So each time any of LAPIS or the docs change?
(This would mean we must also wait below on the LAPIS vs docker image

Reply:
Both, The LAPIS v2 workflow and the Docs workflow will run on main. So it will always be triggered on main anyway.

The last step in this workflow must assume that both Docker images have been built. 
* `workflow_run LAPIS v2` unsures that the LAPIS image is ready
* the `wait-on-check-action` ensures that the Docs image is ready.